app/views/quotes/show_quote.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "daily_quote" %>
```

Comment:
No, I do not want the quote to show on a page by itself. A partial file was created so the page could be included as part of another view.  (_quote.html.erb)